tests/ops/qubit/test_attributes.py

```diff
@@ -151,6 +151,15 @@ class TestSupportsBroadcasting:
     """Test that all operations in the ``supports_broadcasting`` attribute
     actually support broadcasting."""
 
+    @pytest.fixture(scope="function", autouse=False)
+    def capture_warnings(self, recwarn):
+        """Capture warnings."""
+        yield
+        if len(recwarn) > 0:
+            for w in recwarn:
+                assert isinstance(w.message, qml.PennyLaneDeprecationWarning)
+                assert "QubitStateVector is deprecated" in str(w.message)
+
```

Comment:
I'd also prefer if we are bit more specific in our warnings control here.

Reply:
I'm not sure what you mean here. Do you have any examples of what would be good practice? @albi3ro

Reply:
@albi3ro